data/assets/scene/digitaluniverse/constellations.asset

```diff
@@ -1,24 +1,32 @@
 local speck = asset.syncedResource({
-  Name = "Constellation Speck Files",
+  Name = "Constellation Files",
   Type = "HttpSynchronization",
-  Identifier = "digitaluniverse_constellations_speck",
-  Version = 2
+  Identifier = "digitaluniverse_constellations_data",
+  Version = 1
 })
 
+local zodiacs = {
+  "Cancer", "Taurus", "Pisces", "Aries", "Libra", "Aquarius", "Capricornus", "Scorpius",
+  "Virgo", "Sagittarius", "Gemini", "Leo"
+}
+
 local constellationsExtragalactic = {
   Identifier = "ConstellationsExtragalactic",
   Renderable = {
-    Type = "RenderableDUMeshes",
+    Type = "RenderableConstellationLines",
     Enabled = false,
     Opacity = 0.4,
     File = speck .. "constellationsEXGAL.speck",
     LabelFile = speck .. "constellationsEXGAL.label",
+    ConstellationNamesFile = speck .. "constellations.dat",
     TextColor = { 0.8, 0.8, 0.8 },
     TextOpacity = 0.4,
     TextSize = 20.0,
     TextMinMaxSize = { 20, 30 },
-    MeshColor = { { 0.6, 0.4, 0.4 }, { 0.8, 0.0, 0.0 }, { 0.0, 0.3, 0.8 } },
-    Unit = "Mpc"
+    ConstellationColor = { { 0.6, 0.4, 0.4 }, { 0.8, 0.0, 0.0 }, { 0.0, 0.3, 0.8 } },
```

Comment:
"LineColors"?

I'm also a bit confused about how the multiple colors are handled. Could we update the property info with more details? :)

Reply:
The constellations can be grouped and each group can have a different color. For example, the zodiacs currently have a red color, Ursa Major and Orion have a blue color while the rest have a fainter red/orange color. I will try and update the info to explain this better.